eclass/elisp-common.eclass

```diff
@@ -457,7 +457,7 @@ elisp-site-regen() {
 		# was actually no change.
 		# A case is a remerge where we have doubled output.
 		rm -f "${T}"/site-gentoo.el
-		eend
+		eend $?
```

Comment:
Actually, should be `eend 0` here, because it has no bad consequences if the `rm` fails.

Reply:
WFM. Could you push it yourself, so I'll just remove it as part of rebase?

Reply:
Done.